emukit/quadrature/methods/warped_bq_model.py

```diff
@@ -133,3 +134,12 @@ def integrate(self) -> Tuple[float, float]:
         :returns: Estimator of integral and its variance.
         """
         raise NotImplemented
+
+    @staticmethod
+    def _symmetrize(A: np.ndarray) -> np.ndarray:
+        """Symmetrize a matrix.
```

Comment:
Can you add a brief explanation what `to symmetrize` means?

Reply:
The method should probably be called `symmetrize_matrix`, but I'll remove it anyways for now. It means 0.5  (A + A') where A is the matrix.